### DISCUSSION TITLE
Our last Sorobanathon entries!

### DISCUSSION THREAD

**Discussion by DebravdH** [stellar/sorobanathon#166] [General]
We accidentally missed the prolog, so today we decided to act it out for this one: 
https://litemint.com/items/GAYG2JPZCMPWFXVPWHAAM22JDNAQNHKBS4X42I7J7QGHGBHBJMHSUKFY/BeeRoscoead

5 and 6 we themed together since they kind of happen in the same scene/room/island. 
This is where Bee and Roscoe fight Zebulon. (Story 5)
https://litemint.com/items/GACYDCYHFZXZABFSRZLNGDWD6ZWIRSYBWE3Y6BZIKQXVR4ROOHHSHVNB/nothimagain
And this is story 6 where they meet the luck fish. (We had a lot of fun with this one) 
https://litemint.com/items/GAGBW77E3UJWDTHPV3DONGCNRLKLHH7U3EOC2SW53E3EJ7CD72TZ76DD/Luckfish

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 3900 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
